codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -101,6 +101,8 @@ organisations->organizations
 organise->organize
 organised->organized
 plagiarise->plagiarize
+polarise->polarize
+polarised->polarized
```

Comment:
Also the valid ones from this list? http://app.aspell.net/lookup?dict=en_GB-ise&words=polarise%0D%0Apolariser%0D%0Apolarising%0D%0Apolarised%0D%0Apolarises